src/components/manage/AdminUser/AdminUser.constant.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * CORTX-CSM: CORTX Management web.
+ * Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com.
+ */
+
+export interface IUserDetail {
```

Comment:
have a separate file for constant which would hold all the constants of that implementation, and for the interface you could have multiple files depending upon feature or your use case.

Reply:
Separated the content in LrAdminUser.constant and LrAdminUser.model files.